docs/data/date-pickers/date-picker/SubComponentsPickers.js

```diff
@@ -17,19 +17,19 @@ export default function SubComponentsPickers() {
     <LocalizationProvider dateAdapter={AdapterDayjs}>
       <Grid container spacing={3}>
         <Grid item xs={12} md={6}>
-          <CalendarPicker date={date} onChange={(newDate) => setDate(newDate)} />
+          <CalendarPicker value={date} onChange={(newDate) => setDate(newDate)} />
```

Comment:
For naming consistency in demo?

```suggestion
          <CalendarPicker value={value} onChange={(newValue) => setValue(newValue)} />
```